### PR TITLE
Expose supported encryption types and their default

### PR DESCRIPTION
**Pull request by mvollmer**
For the benefit of UIs such as Cockpit.

### CODE REVIEW

**Comment by vojtechtrefny** [comment]
Jenkins, ok to test.

File: src/udiskslinuxmanager.c [hunk 1]
```diff
@@ -144,6 +145,8 @@ udisks_linux_manager_set_property (GObject      *object,
     }
 }
 
+static const gchar *supported_encryption_types[] = { "luks1", "luks2", NULL };
```
Comment (mvollmer) [inline src/udiskslinuxmanager.c]: We should probably find a better place for this.

Comment (mvollmer) [reply src/udiskslinuxmanager.c]: Done.

**Comment by mvollmer** [comment]
> Could you please reword the commit messages to include name of the object/source instead of the FIXUP string? Just a cosmetic thing but it hits me in my eyes :-)

Of course! :-)  The FIXUP commits are meant to be squashed into the main commit, but I kept it separate to avoid force pushing which might have disturbed the review.

**Comment by tbzatek** [comment]
> Of course! :-) The FIXUP commits are meant to be squashed into the main commit, but I kept it separate to avoid force pushing which might have disturbed the review.

Hah, you've got a point! :+1:  Don't worry about disturbing the review, this is a simple one and sometimes it's easy to read through squashed changed.

Anyway, looks good now, thanks!

**Pull request merged**